juriscraper/opinions/united_states/state/nh_p.py

```diff
@@ -76,31 +72,28 @@ def _process_html(self) -> None:
                 0
             ].split("//")[1]
 
-            if "," not in case["title"]:
-                logger.info("Skipping case with title %s", case["title"])
-                continue
-
             # "title" format changes since 2024, where a citation string
             # replaces the docket number, and the docket is in another row
-            docket_or_cite, name = case["title"].split(",", 1)
-            if self.docket_regex.search(docket_or_cite):
-                docket = docket_or_cite
-                neutral_citation = ""
-            else:
-                neutral_citation = docket_or_cite
+            title_match = self.title_regex.search(case["title"])
+            if not title_match:
+                logger.warning("Unable to parse title %s", case["title"])
+                continue
+
+            title_fields = title_match.groupdict("")
+            if not title_fields.get("docket"):
                 docket_str = case["fields"]["field_description"][0]["#text"]
-                docket = self.docket_regex.search(docket_str).group(0)
+                title_fields["docket"] = re.search(
+                    r"\d{4}-\d+", docket_str
+                ).group(0)
 
             self.cases.append(
                 {
-                    "name": name.strip(),
-                    "docket": docket.strip(),
                     "date": case["fields"]["field_date_posted"][0],
                     "url": urljoin(
                         "https://www.courts.nh.gov/sites/g/files/ehbemt471/files/",
                         url,
                     ),
-                    "citation": neutral_citation,
+                    **title_fields,
```

Comment:
as much as I appreciate this - I think we should be clear about what is being added.